### DISCUSSION TITLE
Automatic 1.8.2 release production

### DISCUSSION THREAD

**Discussion by agrignard** [gama-platform/gama#85] [Ideas]
**Describe the bug**
Is seems that the automatic release production is broken. The last release available is from 25_021_22 and it now 42 commits ahead from the master

The travis workflow (https://github.com/gama-platform/gama/actions) seems to be called everyday and looks good (green) but the release is not pushed/produced here
https://github.com/gama-platform/gama/releases/tag/1.8.2

**Comment by RoiArthurB** [comment]
Hi,
We run a compilation check on each commit, but GAMA 1.8.2 Alpha Release are only deployed on command (with message command or with GitHub's UI) :) 

I'll rebuild a fresh version for you ;)

**Comment by agrignard** [comment]
thank you @RoiArthurB but instead of having you doing it manually I though the idea was to have an automatic process (maybe not every commit but let's say every day)? Am i wrong? What is the process though github UI to do it so i don't have to bother you too much?

**Comment by RoiArthurB** [comment]
As it is a pretty unstable version, publishing a new version everyday may lead some people to download a broken version of GAMA. So I personally think that, as we use to recommend some people from the mailing-list for instance to try this version, it's better to make some manual release when we want to try a new thing. Also, as developers, the compiled version doesn't provide any advantage over the one in Eclipse; so it's not needed to have a new daily version 🤷🤷🤷

Here's how to re-launch a build : 
![image](https://user-images.githubusercontent.com/16764085/157838470-a18c913d-87b5-4bc0-bf8f-b35b637bc269.png)
(the point 4 is important; otherwise you'll relaunch a workflow but on the master branch (and, then, not make a new release of GAMA 1.8.2 :wink: )

**Comment by hqnghi88** [comment]
I agree with Arthur, exactly what i was writing

**Comment by agrignard** [comment]
Ok thank you Arthur for the release I'll test it. I also understantd now how to generate one (I was indeed missing the point 4 and using the master)

I also agree with you if we have an instable version it's tricky to generate one everyday (unless we clearly specify it's the current version) but that's a long discussion. 

After for me there is 3 kind of user: 
1. the developper that have eclipse install and for them for sure a release is not needed. However those people are not necessarily the one that stress test the platform and find new issues that emerges from specific usage/models
2. The 'expert' user that are kind of always using the last feature of GAMA and often the one  finding new issue(this is one fo the reason of the creation of this issue, I have some colleagues that do not use eclipse (and I cannot force them) but need the last release
3. The classical user that can download the stable one (which start to be very different from the current one but it is how it is)

Thank for the very quick answer !

**Comment by hqnghi88** [comment]
for 2: yes, but this kind of user also should base on a more stable version than "alpha", otherwise there will be many new issue :)

**Comment by RoiArthurB** [comment]
(converting the issue to a discussion as it seems more appropriate to discuss how to release GAMA and for whom :) )
